Healthy/Classes/Extensions/UIButton+Style.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UIButton+Style.swift
+//  Healthy
+//
+//  Created by Ahmed Nasr on 13/05/2023.
+//
+
+import UIKit
+
+
+
+// MARK: - Button style
+//
+extension UIButton {
+    enum ButtonStyle {
+        case primary
+        case secondary
+        case disable
+        
+        var buttonColor: UIColor {
+            switch self {
+            case .primary:
+                return .black
+            case .secondary:
+                return .green
+            case .disable:
+                return .gray
+            }
+        }
+    }
+}
+
+
+
+// MARK: - Apply button style
+//
+extension UIButton {
+    func applyButtonStyle(style: ButtonStyle) {
+        setButtonStyleAttributes(style: style)
+    }
+}
+
+
+
+// MARK: - set button style attributes
```

Comment:
```suggestion
// MARK: - Set button style attributes
```
Sentence should start with a capital letter

Reply:
Done